cmd/cli/install.go

```diff
@@ -53,10 +53,10 @@ well as for adding a Kubernetes Namespace to the list of Namespaces a control
 plane should watch for sidecar injection of Envoy proxies.
 `
 const (
-	defaultCertManager   = "tresor"
-	defaultVaultProtocol = "http"
-	defaultMeshName      = "osm"
-
+	defaultCertManager         = "tresor"
+	defaultVaultProtocol       = "http"
+	defaultMeshName            = "osm"
+	osmControllerLabel         = "osm-controller"
```

Comment:
can you move this tp pkg/constants/constants.go?

Otherwise LGTM!

Reply:
done :)